someip2val/src/lib/broker_feeder/CMakeLists.txt

```diff
@@ -35,10 +35,19 @@ find_program(GRPC_CPP_PLUGIN_PROGRAM_ENV
 )
 get_filename_component(GRPC_CPP_PLUGIN_PROGRAM_ENV "${GRPC_CPP_PLUGIN_PROGRAM_ENV}" ABSOLUTE)
 
+message("--- [broker_feedder] Using protoc: ${PROTOC_PROGRAM_TEMP}")
+message("--- [broker_feedder] Using grpc_cpp_plugin: ${GRPC_CPP_PLUGIN_PROGRAM_ENV}")
+message("--- [broker_feedder] protoc version:")
+execute_process (
+  COMMAND "${PROTOC_PROGRAM_TEMP}" --version
+)
+
 add_library(data_broker_feeder
   STATIC
     data_broker_feeder.cc
     collector_client.cc
+    # logger.cc
```

Comment:
Do we plan to add a logger.cc? Or shall we just remove this comment?

Reply:
I'll remove it, it turned out more complex to do than expected.